docs/source/en/_toctree.yml

```diff
@@ -57,7 +57,7 @@
         title: Summarization
       - local: tasks/multiple_choice
         title: Multiple choice
-      title: Task guides
+      title: Task guides      
```

Comment:
This PR should't touch this

Reply:
I remember that I only add two lines about conditional detr in this file, I do not know why we have these changes...anyway let me manually fix this.